src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  max-width: ($cell-size * $days-in-week) + (2 * $cell-pad) + ($cell-gap * 6);
+  padding: $cell-pad;
+  gap: $cell-gap;
+  display: flex;
+  flex-wrap: wrap;
+
+  @each $start-day, $number in $days {
+    &_start-day--#{$start-day} >:first-child {
+      margin-left: ($cell-size + $cell-gap) * $number;
+    }
+  }
+
+  &__day {
+    @include square($cell-size);
+
+    position: relative;
+    border: 1px solid black;
+    background-color: #eee;
+    cursor: pointer;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+      transition-duration: 0.8s;
```

Comment:
1. If you put transition parameters under general selector (not :hover) it will work both ways.
2. 
![image](https://user-images.githubusercontent.com/60885495/125656318-be16286e-fe2d-4b1e-ae85-5a9c9b8647b6.png)

3. Always add rules that you apply transition to:

```suggestion
      transition-duration: transform 0.5s, background-color 0.5s;
```